homeassistant/components/tractive/const.py

```diff
@@ -28,6 +28,7 @@
 TRACKER_POSITION_UPDATED = f"{DOMAIN}_tracker_position_updated"
 TRACKER_SWITCH_STATUS_UPDATED = f"{DOMAIN}_tracker_switch_updated"
 TRACKER_WELLNESS_STATUS_UPDATED = f"{DOMAIN}_tracker_wellness_updated"
+TRACKER_HEALTH_OVERVIEW_UPDATED = f"{DOMAIN}_tracker_health_overview_updated"
```

Comment:
Sensors `minutes_active`, `minutes_rest`, `daily_goal`, `minutes_day_sleep` and `minutes_night_sleep` need to use this prefix, e.g.:

```python
    TractiveSensorEntityDescription(
        key=ATTR_MINUTES_NIGHT_SLEEP,
        translation_key="minutes_night_sleep",
        native_unit_of_measurement=UnitOfTime.MINUTES,
        signal_prefix=TRACKER_HEALTH_OVERVIEW_UPDATED,
        state_class=SensorStateClass.TOTAL,
    ),
```